src/main/res/values/strings.xml

```diff
@@ -280,15 +280,15 @@
     <string name="comment_edit_done">Comment edited. Changes will appear after you refresh.</string>
     <string name="post_submit_title_hint">Title</string>
     <string name="post_submit_subreddit_hint">Subreddit</string>
-    <string name="post_captcha_title">Captcha</string>
+    <string name="post_captcha_title">CAPTCHA</string>
     <string name="post_captcha_submit_button">Submit</string>
     <string name="post_submit_done">Post submitted.</string>
 
     <!-- Bug Reporting -->
 
     <string name="bug_title">An error has occurred.</string>
-    <string name="bug_message">RedReader has encountered an error.\n\nWould you like to send a bug report to the developer? If so, an email will be filled in for you with all the relevant information, and you can inspect the contents before sending.</string>
-    <string name="bug_button_send">Send bug report by email</string>
+    <string name="bug_message">Would you like to send a bug report to the developer? If so, an e-mail with all the relevant info will be filled in for you, and you can inspect it all before sending.</string>
+    <string name="bug_button_send">Send bug report by e-mail</string>
```

Comment:
I generally prefer "email" to "e-mail"

Reply:
Produces errors in translation though. The dash isn't optional in many other languages. Easier to omit when not there than to include in error.

Reply:
The other translations can use "e-mail" if that's more appropriate for those particular languages, for English we should stick with "email" though.

The word "Mail" in those three examples is used in the sense of "message from another user", so yes it is internal reddit mail, but the English translation should stay as "mail".

Reply:
@QuantumBadger It isn't appropriate. I am saying people misunderstand it in translation, to mean "arrived e-mail" and "has e-mail address".

Reply:
We shouldn't modify strings just to provide context for translators, when the context is evident in the UI for the user. It would be better to leave a comment on Weblate for future translators.

Reply:
"E-mail" is also the proper form, consistent with other such words. You may disagree, but people that don't exist. "Email" just recently came into fashion. "E-mail" will be there till it all becomes "mail" and regular mail is either that or "snailmail".

Reply:
I've gone ahead and removed two of these strings as they were unnecessary and mostly debug-related when the app was first written: https://github.com/QuantumBadger/RedReader/commit/f70d8aa7ca9e3e90b1df174eec184992930c6357

The third one ("Mod Mail") is the term Reddit themselves use, although it looks like they actually use one word "Modmail", so feel free to change it to that: https://old.reddit.com/r/modguide/comments/dlc0v9/how_to_use_modmail/

I agree "email" and "e-mail" are both valid, but since (1) "email" is in fashion, and (2), RedReader already uses "email", let's keep it as it was.